### DISCUSSION TITLE
Injecting the latest version with a custom script?

### DISCUSSION THREAD

**Discussion by nielsbasjes** [renovatebot/renovate#10544] [Request Help] [answered]
So I have this strange edge case that Google does not provide the correct tags on the docker images within their mirror.gcr.io registry. See https://issuetracker.google.com/issues/137170318 and my earlier question about this https://github.com/renovatebot/renovate/discussions/10302 .

Funny thing is that 
1. I cannot query the official docker registry from renovate (because of firewall) 
2. when I instruct Google to actually build me an image based on the image I specified that just works (because it runs somewhere else ... different firewall).

Like I said: Very strange edgecase and I do not expect this to be handled by renovate .

Now I'm capable of writing a (bash/curl/jq) script that determines the version from the latest tag on the `mirror.gcr.io` by extracting the `org.opencontainers.image.version` label.

My question is if there is a way for me to "inject" this information (i.e. for a specific dependency I already know the latest version) into renovate?

**Comment by rarkins** [comment, marked as answer]
No, not currently possible. This would need the ability to define custom datasources somehow.
